modules/metering-troubleshooting.adoc

```diff
@@ -10,10 +10,173 @@ A common issue with metering is pods failing to start. Pods might fail to start
 [id="metering-not-enough-compute-resources_{context}"]
 == Not enough compute resources
 
-A common issue when installing or running metering is lack of compute resources. Ensure that metering is allocated the minimum resource requirements described in the installation prerequisites.
+A common issue when installing or running metering is a lack of compute resources. As the cluster grows and more reports are created, the `reporting-operator` pod requires more memory. If memory usage reaches the pod limit, the cluster terminates the pod with an `OOMKilled` status. Ensure that metering is allocated the minimum resource requirements described in the installation prerequisites.
+
+NOTE: CPU usage for the `reporting-operator` pod does not increase as the cluster grows.
```

Comment:
This is probably sufficient, but I think the general detail here is the metering operator does not autoscale the reporting operator based on the load in the cluster.

Reply:
👍 I can add that clarification to the note.